packages/cli/src/index.ts

```diff
@@ -44,6 +44,7 @@ async function runCommand(cmd: CLICommand, flags: yargs.Arguments): Promise<numb
         tables: {
           Commands: [
             ['create', 'Create new tutorial app'],
+            ['eject', 'Eject routes'],
```

Comment:
@AriPerkkio Not sure what you have in mind for the description here. I suppose it can be more descriptive and `Eject routes` also isn't entirely correct. Any thoughts?

Reply:
Maybe we could say that it exposes the astro pages and components that are normally handled by the integrations which gives full control about the Astro app. 

I suppose this is it, so how about:

```suggestion
            ['eject', 'Move all default pages and components in your project, providing full control over the Astro app']
```

Reply:
Yep, whatever you feel makes more sense 👍 Feel free to just push into this branch. I'd say `... into your project,...` tho.

Reply:
Sounds good, let's use this.